### PR TITLE
Node scoped device GUID and use blake2b instead of md5

### PR DESCRIPTION
**Pull request by weihanglo**
Address https://github.com/harvester/harvester/issues/1195 and https://github.com/harvester/harvester/issues/1196

- Add `nodeName` to hash function, so the same devices on different nodes are treated as different block device.
- Use blake2b instead of md5 to produce hash value. I am not sure whether 16-bytes digest size is ennough, but there is a limit that a label must not be more than 63 characters, so we cannot use the recommended size (32-bytes) of blake2b hash.

### CODE REVIEW

File: harvester-node-disk-manager/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
```
Comment (guangbochen) [inline harvester-node-disk-manager/Chart.yaml:1]: ditto

File: harvester-node-disk-manager/.helmignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (guangbochen) [inline harvester-node-disk-manager/.helmignore:1]: why do we need this file?

File: harvester-node-disk-manager/Chart.yaml (hunk 1 above)
Comment (weihanglo) [reply harvester-node-disk-manager/Chart.yaml:1]: A mistake. Sorry I'll remove them.

File: harvester-node-disk-manager/.helmignore (hunk 2 above)
Comment (weihanglo) [reply harvester-node-disk-manager/.helmignore]: Removed. Sorry for the inconvenience.

**Review by guangbochen — APPROVED** [review]
LGTM

**Review by bk201 — APPROVED** [review]
LGTM!

**Pull request merged**